src/components/batch-selection/batch-selection.component.js

```diff
@@ -11,6 +11,17 @@ const BatchSelection = ({
   selectedCount,
   hidden
 }) => {
+  const getTextForCount = count => I18n.t(
+    'batch_selection.selected',
+    {
+      count: Number(count),
+      defaultValue: {
```

Comment:
If the options are the same you don't need seperate keys for `one` and `other`. 

```diff
+ defaultValue: 'selected'
```

Reply:
fixed